app/models/reservation.rb

```diff
@@ -1,6 +1,7 @@
 class Reservation < ApplicationRecord
   belongs_to :space
   belongs_to :user
+  has_one_attached :image
```

Comment:
Should come from space rather than reservation itself; likely unnecessary?

Reply:
true, was early try. ill check